winterfell/models/userValues.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+var mongoose = require('mongoose');
+var User = require('./user');
+var Schema = mongoose.Schema;
+
+var UserValueSchema = new Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
+  },
+  values: Schema.Types.Mixed
+}, { timestamps: true });
```

Comment:
with this, Schema will automatically create a `createdAt` and `updatedAt` fields for you.
Because these default values are camelcase and using snakecase for db fields in the first commit was a bit awkward, let's just go ahead and use camelcase all around our casebase.

Maybe at least constants can still be CAPITAL_SNAKE_CASE